assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/bootstrap-configs.tf

```diff
@@ -0,0 +1,40 @@
+locals {
+  controller_bootstrap_tokens = [
+    for index in range(var.controller_count) : {
```

Comment:
I'd use single token for all controllers, to support #28. And also to not make controllers much different from worker pools.

Reply:
The tokens are not assigned to worker pools, but to worker. Token per node.

Reply:
I'm saying we should use token per worker pool, so all worker pools use the same configuration.

Reply:
But that causes the entire pool to be recreated thus losing data. How do we mitigate that?

Reply:
> Note, that we will replace nodes, as part of upgrade process, as we discussed some time ago.

That was only for ephemeral nodes. What about the nodes with stateful apps?